docs/source/private_data_tutorial.rst

```diff
@@ -731,16 +746,29 @@ When successful, should see something similar to the following result:
 Query private data Org2 is not authorized to
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Peers in Org2 do not have the Org1 asset1 ``appraisedValue`` private data in their side database.
-When they try to query for this data, they get back a hash of the key matching
-the public state but will not have the private state.
+Peers in Org2 do not have the ``appraisedValue`` of the asset1 owned by Org2 in their
```

Comment:
On reading this sentence again, I think the line is confusing since 

 asset1 is still owned by Org1 (transfer hasn't happened yet) ... the peer is of course owned by Org2.

Rephrase, or perhaps delete the "owned" part

Reply:
Rephrasing: 
Because the asset was created by Org1, peers in Org2 do not store the
``appraisedValue`` associated with ``asset1``. Run the following command to
demonstrate that the asset's ``appraisedValue`` is not stored in the
``Org2MSPPrivateCollection`` on the Org2 peer:

Reply:
sweet